src/ApiCheck/Baseline/BaselineVisibility.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace ApiCheck.Baseline
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum BaselineVisibility
+    {
+        Public,
+        Protected,
+        Internal,
```

Comment:
Its interesting to have the full classification for testing purposes. It's equally important to make sure that the tool classifies public, protected, etc types and members as public, protected, etc. As it is to make sure that something internal or private does not get classified with a different visibility by mistake.

The tool has other mechanisms in place to filter the set of types included in the generated API listing that filter any unwanted type (like anything non public/protected or anything on the .Internal namespace).